lib/utils.js

```diff
@@ -177,8 +177,8 @@ const hasPullRequestBeenApproved = async (context, pullRequestNumber) => {
  * @param {Number} pullRequestNumber
  */
 const doesPullRequestHaveChangesRequested = async (
-  context,
-  pullRequestNumber
+    context,
+    pullRequestNumber
```

Comment:
Why this change?

Reply:
I think it's due to the eslint errors I was getting locally.